### ISSUE TITLE
Need policy clarification on documentation licensing

### ISSUE THREAD

**Issue by jberkus** [cncf/foundation#230]
Our charter has this confusing language about documentation licensing:

```
(e) All projects evaluated for inclusion in the CNCF shall be completely licensed under an OSI-approved open source license. If the license for a project included in CNCF is not Apache License, Version 2.0, approval of the Governing Board shall be required.

(f) All documentation will be received and made available by the CNCF under the Creative Commons Attribution 4.0 International License.
```

Thus all project materials must be available under the AL2, but all documentation must be available under CCBY.  This language has always been contradictory.

Most projects have taken this to mean that documentation, particularly documentation that is embedded in code repos, *may* be licensed under either AL2 or CCBY, as circumstances warrant.  However, recently, some folks involved in the CNCF ecosystem have taken this to mean that all documentation, even a README, *must* be licensed CCBY no matter what, even if that requires adding a separate license header to every file or even portions of a file.  This interpretation of the policy appears to be based on the mistaken belief that the AL2 cannot legally be used for documentation (something the Apache Project would be very surprised to hear).

And it's [stalled us in Contributor Strategy](https://github.com/cncf/project-template/pull/12) based on "what guidance do we put in the templates regarding license".

My proposal is that the GB clarify that projects *must* use AL2 for code, and *can* use either CCBY or AL2 for documentation and content (like websites).

**Comment by amye** [comment]
This is currently scheduled for review in the June 8th meeting of the Legal Committee.

**Comment by amye** [comment]
Approved by Legal Committee and GB as of a vote on 8-31, minutes will be approved as of the next governing board meeting

**Comment by craigbox** [comment]
I don't see any mention of this in the [Sep 2023](https://www.cncf.io/wp-content/uploads/2023/10/CNCF-Governing-Board-Meeting-Minutes-2023-09-14-PUBLIC.pdf) or [Dec 2023](https://www.cncf.io/wp-content/uploads/2024/01/CNCF-Governing-Board-Meeting-Minutes-2023-12-14-PUBLIC.pdf) meeting minutes, and the charter hasn't been changed. Can I check in on the next steps here please @amye?

**Comment by krook** [comment]
This is what was approved by an email vote of the Governing Board on August 31, 2023:

> RESOLVED, that the Governing Board hereby approves a license exception to permit CNCF project source code repositories’ documentation files to be published under either Apache-2.0 or CC-BY-4.0.

To answer a related question about relicensing existing APL2 docs as CC-BY-4.0 if desired, the answer from @joannalee333 is that they can be relicensed as long as each author of the docs approves the change of license. Relicensing documentation to CC-BY4.0 doesn't require any other approval of foundation staff or the GB.
